crates/amaru-consensus/src/consensus/mod.rs

```diff
@@ -40,6 +43,12 @@ pub mod header_validation;
 pub mod nonce;
 pub mod store;
 
+#[derive(Clone)]
+pub enum PullEvent {
+    RollForward(Peer, Point, Vec<u8>, Span),
+    Rollback(Peer, Point, Span),
+}
```

Comment:
Moved this here from the ouroboros crate; it makes slightly more sense since this is the actual place where this is used.